docs/001_develop/03_client-capabilities/005_grids/002_entity-manager/01_entity-manager-grid.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Grid configuration'
+sidebar_label: 'Grid configuration'
+id: entity-manager-grid
+keywords: [grid configuration, entity-management, entity manager, table setup, customization, data grid, UI, frontend, grid options, user interface, data display, micro-front-ends]
+tags:
+  - entity management
+  - frontend
+  - micro-front-ends
+  - profile management
+  - ui
+  - user management
+  - web
+  - grid configuration
+  - entity manager
+  - table setup
+  - customization
+  - data grid
+  - grid options
+  - user interface
+  - data display
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+The Grid is a key component of the **Entity Management**, enabling you to display and manage data from back-end resources. It offers a flexible, structured layout where each row represents an entity, and each column represents an attribute of that entity. The grid can be easily configured to pull data from specified back-end resources and is automatically populated with relevant information.
+
+This section explains how to configure and customize the grid to suit your application's needs.
+
+## Examples 
+
```

Comment:
I think this title is unnecessary in the case where the entire file is just a set of examples, same for the other pages too

Reply:
Done in [chore: addressed PR feedback PTL-1513](https://github.com/genesiscommunitysuccess/docs/pull/1951/commits/8e0a48c144037b54d8e8856d88f850cd48e063d9)